e2e/keywords/longhorn.resource

```diff
@@ -43,3 +44,9 @@ Check Longhorn workload pods ${condition} annotated with ${key}
     Run Keyword IF    '${condition}' == 'not'    Should Not Be True    ${is_annotated}
     ...    ELSE IF    '${condition}' == 'is'    Should Be True    ${is_annotated}
     ...    ELSE    Fail    Invalid condition ${condition}
+
+Uninstall Longhorn ${LONGHORN_BRANCH} by ${INSTALL_METHOD}
```

Comment:
We can use the `INSTALL_METHOD` environment variable to determine which class to instantiate, eliminating the need to explicitly specify `Uninstall Longhorn by XXX`. WDYT?

Reply:
In `libs/longhorn_deploy/lognhorn.py` added below part because Jenkins did not have `LONGHORN_INSTALL_METHOD` on e2e pipeline, if run on local and using helm, we need to set set `LONGHORN_INSTALL_METHOD` to `helm-chart` first
```
_method = os.getenv("LONGHORN_INSTALL_METHOD", "manifest-file")
```